canonical-web-and-design/ubuntu.com
# Ubuntu AWS and Ubuntu pro pages

## Done
Build of the AWS and AWS pro pages

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to /aws and check it matches the [copy doc](https://docs.google.com/document/d/1lkH-MSPmgjCMP4gI0FW29YIKVDvBuddxA63iQhmsYIU/edit?ts=5de63dcb#) and the [design](https://usercontent.irccloud-cdn.com/file/mjMJMvDg/image.png)
- Go to /aws/pro and check it matches the [copy doc](https://docs.google.com/document/d/1pLRWJZSpE04nYt-yHBtCjxGI7VZsigcP2UJIBmR8i-0/edit#)
- Go to /aws/contact-us and check to works
- Check the thank you page at /aws/thank-you looks ok

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2030
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2032
